supporting_content/metrics/pricingquantity.md

```diff
@@ -0,0 +1,93 @@
+# Column: PricingQuantity
+
+## Example treatment for non-metered charges (e.g. Tax Treatment)
+
+Example A
+| Description          | Pricing Quantity          | Unit Price   | Total     |
+|:---------------------|:--------------------------|:-------------|-----------|
+| Compute Usage        | 100                       | 1.0          | $100      |
+| Tax on Compute Usage | 100                       | 0.2          | $20       |
+| TOTAL                | 200                       | 1.2          | $120      |
+NOTE: Double counting of hours
+
+Example B
+| Description          | Pricing Quantity          | Unit Price   | Total     |
+|:---------------------|:--------------------------|:-------------|-----------|
+| Compute Usage        | 100                       | 1.0          | $100      |
+| Tax on Compute Usage | 1                         | 20           | $20       |
+| TOTAL                | 101                       | 21           | $120      |
```

Comment:
Unit Price is not a column that can be summed because by doing so, the result violates this clause of the column definition "When unit prices are not null, multiplying PricingQuantity by a unit price MUST produce a result equal to the corresponding cost metric."  I see the below note and the one in Example A, but my recommendation for these examples would be to either calculated Unit Price (weighted average) on the TOTAL row or leave it NULL.

Should the total column be [Billed Cost](https://github.com/FinOps-Open-Cost-and-Usage-Spec/FOCUS_Spec/blob/working_draft/specification/metrics/billedcost.md)/BilledCost?

Reply:
We assumed that this would be addressed in detail as part of the SKU/SKU Price discussion. This section contains notes from our discussion, and we marked Example 3 as the preferred/suggested one. We made some changes based on your suggestions and moved it to the discussion/scratch space chapter.

Please let me know if I can close this conversation.
Thanks!